### DISCUSSION TITLE
cannot be used directly

### DISCUSSION THREAD

**Discussion by LLLLLLLccWWW** [luicfrr/react-native-vision-camera-face-detector#57] [Q&A]
Can this be used directly after downloading? In addition to yarn add react-native-vision-camera-face-detector and npm i react-native-worklets-core, is there anything else that needs to be changed?

**Comment by luicfrr** [comment]
Did you checked instalation guide on `README`?

**Reply by LLLLLLLccWWW** [reply]
Yes, I first yarn add react-native-vision-camera-face-detector and add react-native-worklets-core plugin to babel.config.js according to the README, but after execution, D:\react-native-vision appears. -camera-face-detector\android>npx react-native run-android

  WARNING: You should run npx react-native@latest to ensure you're always using the most current version of the CLI. NPX has cached version (0.73.8) != current release (0.74.1)

* daemon not running; starting now at tcp:5037
* daemon started successfully
info Launching emulator...
error Failed to launch emulator. Reason: The emulator (INFO | Storing crashdata in: C:\Users\User\AppData\Local\Temp\\AndroidEmulator\emu-crash-34.2.13.db, detection is enabled for process: 19552 ) quit before it finished opening. You can try starting the emulator manually from the terminal with: C:\Users\User\AppData\Local\Android\Sdk/emulator/emulator @INFO | Storing crashdata in: C:\Users\User \AppData\Local\Temp\\AndroidEmulator\emu-crash-34.2.13.db, detection is enabled for process: 19552.
warn Please launch an emulator manually or connect a device. Otherwise app may fail to launch.
info Installing the app...
Starting a Gradle Daemon, 4 incompatible and 1 stopped Daemons could not be reused, use --status for details

info 💡 Tip: Make sure that you have set up your development environment correctly, by running npx react-native doctor. To read more about doctor command visit: https://github.com/react-native-community/cli/blob/ main/packages/cli-doctor/README.md#doctor
May I ask what steps I missed or what I did wrong?
